File: Ent_UsAdmin/nombregrupo.vb
Public Class nombregrupo

  
    Dim tiempo As Integer = 0
    Dim opacidad As Double = 1
    Private Sub Timer1_Tick(sender As Object, e As EventArgs) Handles Timer1.Tick
        Timer1.Interval = 100
        If tiempo < 10 Then
            Me.Size = New Size(Me.Width + tiempo, Me.Height + tiempo)
            opacidad = opacidad - 0.05
            Me.Opacity = opacidad
            tiempo += 1

        Else
            Timer1.Enabled = False
            Me.Close()

        End If
    End Sub

    Private Sub nombregrupo_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Me.Location = New Point(usuarios.Location.X + usuarios.Combogrupo.Location.X, usuarios.Location.Y + usuarios.Combogrupo.Location.Y + usuarios.Panel1.Height)
        Me.Size = New Size(Me.Width - 80, Me.Height - 80)
        Timer1.Enabled = True
        Me.TopMost = True
    End Sub

    Private Sub nombregrupo_SizeChanged(sender As Object, e As EventArgs) Handles Me.SizeChanged
        MonoFlat_HeaderLabel1.Location = New Point((Me.Width / 2) - (MonoFlat_HeaderLabel1.Width / 2), (Me.Height / 2) - (MonoFlat_HeaderLabel1.Height / 2))
    End Sub
End Class

File: Ent_UsAdmin/Catalogos/nombregrupo.cs
using System;
using System.Drawing;

namespace ConfiaAdmin
{
    public partial class nombregrupo
    {


        private int tiempo = 0;
        private double opacidad = 1d;

        public nombregrupo()
        {
            InitializeComponent();
        }
        private void Timer1_Tick(object sender, EventArgs e)
        {
            Timer1.Interval = 100;
            if (tiempo < 10)
            {
                Size = new Size(Width + tiempo, Height + tiempo);
                opacidad = opacidad - 0.05d;
                Opacity = opacidad;
                tiempo += 1;
            }

            else
            {
                Timer1.Enabled = false;
                Close();

            }
        }

        private void nombregrupo_Load(object sender, EventArgs e)
        {
            Location = new Point(My.MyProject.Forms.usuarios.Location.X + My.MyProject.Forms.usuarios.Combogrupo.Location.X, My.MyProject.Forms.usuarios.Location.Y + My.MyProject.Forms.usuarios.Combogrupo.Location.Y + My.MyProject.Forms.usuarios.Panel1.Height);
            Size = new Size(Width - 80, Height - 80);
            Timer1.Enabled = true;
            TopMost = true;
        }

        private void nombregrupo_SizeChanged(object sender, EventArgs e)
        {
            MonoFlat_HeaderLabel1.Location = new Point((int)Math.Round(Width / 2d - MonoFlat_HeaderLabel1.Width / 2d), (int)Math.Round(Height / 2d - MonoFlat_HeaderLabel1.Height / 2d));
        }
    }
}